goldens/cdk/menu/index.api.md

```diff
@@ -273,15 +274,6 @@ export type ContextMenuCoordinates = {
     y: number;
 };
 
-// @public
-export class ContextMenuTracker {
```

Comment:
We can't just rename this outside of a breaking change window. You have to re-export the `MenuTracker` under its own name until we can make the breaking change.